doc/superadmin_repos.md

```diff
@@ -0,0 +1,16 @@
+#YUM repos
+YUM repos page is accessible from the menu on the left side. It is shown in the image below.
```

Comment:
I propose to add an explanation what a yum repo is and which are the main attributes.